### PR TITLE
[Bromley] ask for the current bin count when modifying a subscription

### PR DESCRIPTION
**Pull request by struan**
Brings the modify screen in line with new and renew in asking for new bins.

Also refactors the renew/modify templates to reduce shared code.

Fixes mysociety/societyworks#2475

[skip changelog]

### CODE REVIEW

File: templates/web/base/waste/garden/_bin_quantities.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
+    <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
```
Comment (dracos) [inline templates/web/base/waste/garden/_bin_quantities.html:1]: Can this template also be used in subscribe_details.html?

**Review by dracos — CHANGES_REQUESTED** [review]
Few small things

File: templates/web/base/waste/summary.html [hunk 2]
```diff
@@ -17,7 +17,7 @@ <h1 class="govuk-heading-xl">[% title %]</h1>
     <dd class="govuk-summary-list__value">
     </dd>
     <dd class="govuk-summary-list__actions">
-    <form method="post">
+    <form method="post" id="change_answers">
```
Comment (dracos) [inline templates/web/base/waste/summary.html:20]: If you didn't want to add a special ID just for test and you know what `step1` is, I think submit_form_ok({ with_fields => { goto => [step1 value] } }) would also work.

**Review by dracos — CHANGES_REQUESTED** [review]
Left in a debug line is all

**Review by dracos — APPROVED** [review]

**Comment by struan** [comment]
This is awaiting someone at Bromley to test it before going live so I guess it might need a prod: https://mysocietysupport.freshdesk.com/a/tickets/1299

**Pull request merged**